docs/times/11_11.json

```diff
@@ -2,9 +2,17 @@
   {
```

Comment:
Thanks. I do not think the first one is actually a time quote, but rather a train number. Since your second suggestion is great, I think we should add only that.

Reply:
Ah, I understand now: a) this showcases 'R' [a succinct and elegant conversion, indeed], b) the quote must use an actual 'time' [a fascinating requirement], and c) not all 'times' are represented [@20% are not? {a future challenge?}]

So, I edited the CSV, created the JSON from that (first time I used R, let alone tibbled in the tidyverse, ha!), then updated the pull request with both files.

Thank you for being so positive and helpful.